ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -153,8 +153,8 @@
           suffix: "{{ suffix }}"
           env: "{{ env }}"
       data:
-        username: "{{ quay_username | b64encode }}"
-        password: "{{ quay_password | b64encode }}"
+        username: "{{ iib_quay_username | b64encode }}"
+        password: "{{ iib_quay_password | b64encode }}"
```

Comment:
These were renamed across all the `secret-vars.yml` files to avoid confusion with `quay_oauth_token`.